### PR TITLE
LPS-129452 Always call onSelect if the assignee input changes

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/reassign/bulk/select-assignees-step/SelectAssigneesStepTable.es.js [hunk 1]
```diff
@@ -30,27 +30,40 @@ const Item = ({
...
-			filteredTasks.push({
-				assigneeId: newAssignee.id,
-				workflowTaskId: id,
-			});
+		let workflowTasks = reassignedTasks;
+
+		const currentTask = workflowTasks.find(
+			({workflowTaskId}) => workflowTaskId === id
+		);
+
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/reassign/bulk/select-assignees-step/SelectAssigneesStepTable.es.js:47]: Hey @naraandrade,

if possible, can you rewrite this logic to reduce the complexity of this code chunk, lines[48-61]? Thx

**Review by rafaprax — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/reassign/bulk/select-assignees-step/SelectAssigneesStepTable.es.js (hunk 1 above)
Comment (naraandrade) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/reassign/bulk/select-assignees-step/SelectAssigneesStepTable.es.js:47]: @rafaprax  I'll do it, thanks :)

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Review by ramonmsp — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:forward

**Comment by rafaprax** [comment]
Hey @rodrigocunhaa , 

If this pr fails due to unrelated errors again, could you forward directly to BChan?

/cc @naraandrade

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
Manually sent here: https://github.com/brianchandotcom/liferay-portal/pull/100862

**Pull request closed without merging**